cmd/root.go

```diff
@@ -21,23 +38,18 @@ func RootCmd() *cobra.Command {
 	}
 	rootCmd.PersistentFlags().BoolP(cobraext.VerboseFlagName, "v", false, cobraext.VerboseFlagDescription)
 
-	rootCmd.AddCommand(
-		setupBuildCommand(),
-		setupCheckCommand(),
-		setupCleanCommand(),
-		setupExportCommand(),
-		setupInstallCommand(),
-		setupStackCommand(),
-		setupFormatCommand(),
-		setupLintCommand(),
-		setupPromoteCommand(),
-		setupPublishCommand(),
-		setupTestCommand(),
-		setupUninstallCommand(),
-		setupVersionCommand())
+	for _, cmd := range commands {
```

Comment:
nit: won't such form work here?

`rootCmd.AddCommand(commands...)` ?

Reply:
Oh wait, this is is not possible because we're not adding `commands...`. We're actually adding `cmd.Command` for each `cmd` in `commands`.